go/vt/sqlparser/parse_test.go

```diff
@@ -648,8 +648,22 @@ func TestValid(t *testing.T) {
 		input:  "analyze table a",
 		output: "alter table a",
 	}, {
-		input:  "show foobar",
-		output: "other",
+		input:  "show databases",
+		output: "show databases",
+	}, {
+		input:  "show tables",
+		output: "show tables",
+	}, {
+		input:  "show vitess_shards",
+		output: "show shards",
+	}, {
+		input:  "show create database",
+		output: "show unsupported",
+
+		//	This is now a syntax error.
```

Comment:
Actually as discussed my goal is to make this not be a syntax error any more once I get the ForceEOF stuff working.